### PR TITLE
Added Sample for RotatedListView with RotatedItems 

### PR DESCRIPTION
**Pull request by NicolasChampagne**
Listview starts from the bottom and new items are added to the bottom first

## PR Type
What kind of change does this PR introduce?
- Refactoring (no functional changes, no api changes)
- Documentation content changes

## What is the current behavior?
No sample for rotatedlistview

## What is the new behavior?
Sample for rotatedlistview

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal)

Internal Issue (If applicable):
https://nventive.visualstudio.com/Umbrella/_workitems/edit/158082

### CODE REVIEW

**Comment by jeromelaban** [comment]
The updated Xamarin.UITest now allows this sample to be tested

File: src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ListView/RotatedListView_WithRotatedItems.xaml [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+			 xmlns:controls="using:Uno.UI.Samples.Controls"
+			 xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:xamarin="http://uno.ui/xamarin"
+			 mc:Ignorable="d xamarin"
+			 d:DesignHeight="300"
+			 d:DesignWidth="400">
+
+	<UserControl.Resources>
+		<Style TargetType="ListViewItem"
```
Comment (carldebilly) [inline src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ListView/RotatedListView_WithRotatedItems.xaml]: You should try to reduce the template/styles to the minimum required to illustrate your sample.

Comment (NicolasChampagne) [reply src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ListView/RotatedListView_WithRotatedItems.xaml]: @carldebilly This is a sample to illustrate a practical uses of the rotatedListView, and not a sample to reproduce an issue, should I still strip away its styling?

Comment (carldebilly) [reply src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ListView/RotatedListView_WithRotatedItems.xaml]: It depends if the content of the template is important to what the sample is trying to illustrate.

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ListViewTests/UnoSamples_Tests.ListView.cs [hunk 2]
```diff
@@ -15,6 +15,15 @@ namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.ListViewTests
 	public partial class ListViewTests_Tests : SampleControlUITestBase
 	{
 		[Test]
+		[Ignore("Not available yet")]
+		public void RotatedListView_AddsToBottom()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Controls.ListView.RotatedListView_WithRotatedItems");
+
+			//Rotated ListView items can't be properly tests until https://github.com/xamarin/Xamarin.Forms/issues/2496 is fixed.
```
Comment (MatFillion) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ListViewTests/UnoSamples_Tests.ListView.cs]:
```suggestion
			//Rotated ListView items can't be properly tested until https://github.com/xamarin/Xamarin.Forms/issues/2496 is fixed.
```

**Pull request merged**